frontend/src/components/Logs/TableView/config.ts

```diff
@@ -14,12 +14,12 @@ export function getDefaultCellStyle(isDarkMode?: boolean): CSSProperties {
 		lineHeight: '18px',
 		letterSpacing: '-0.07px',
 		marginBottom: '0px',
+		minWidth: '10rem',
```

Comment:
??

Reply:
This is not super important, but it looks cleaner with minWidth and also we have a horizontal scroll, so there will be no problem. 

With MinWidth: 

https://github.com/SigNoz/signoz/assets/26198062/e8b1715c-df65-4de0-a0fb-4d03b9b23b99

Without minWidth:

https://github.com/SigNoz/signoz/assets/26198062/a11b8376-0edc-49f1-8430-a7aeecac779f